src/components/Vote.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+import { patchVote } from "../api";
+
+export const Vote = ({review_id, incVotes, setIncVotes}) => {
+    const [err, setErr] = useState(null);
+
+    const incrementVote = (vote) => {
+        if ((vote === 1 && incVotes !== 1) || (vote === -1 && incVotes !== -1)) {
```

Comment:
You do not need to double up the code here, if you are using an on click to invoke this function with either 1 or -1 your back end should be able to handle those numbers without this conditional logic. 

The incVotes and setIncVotes are surplus to requirements, this component is handling if a single number is being handed to the api.